README.md

```diff
@@ -1 +1,74 @@
-# Hello-Microverse-Project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Hello Microverse Project</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello_Microverse] <a name="about-project"></a>
+
+**[Hello_Microverse]** is a project where I set up a "Hello world" repository. No complex coding is required for it. The goal here is to master all of the tools and best practices learned before. I will be using them in all Microverse projects and most likely in future jobs as well, so it is important to know them!
+
+## 🛠 Built With <a name="built-with"></a>
+HTML, CSS
+
+### Tech Stack <a name="tech-stack"></a>
+Linters
+<details>
+  <summary>Linters</summary>
+  <summary>Client</summary>
+  <ul>
+    <li><a>HTML</a></li>
+    <li><a>CSS</a></li>
+  </ul>
+</details>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
Great work @aima98  and very good description of the project only one small thing to add id steps to setup and install your project if someone wants to use it locally something basic but can be very handy for a beginner.